pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -197,9 +200,151 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 		}
 	}
 
+	//Begin AMQ resource reconcile
+	if instance.Spec.UseMessageBroker {
+		//check that address space exists
+		addressSpace := newAddressSpace(instance)
+
+		// Set UnifiedPushServer instance as the owner and controller
+		if err := controllerutil.SetControllerReference(instance, addressSpace, r.scheme); err != nil {
+			return reconcile.Result{}, err
+		}
+
+		foundAddressSpace := &enmassev1beta.AddressSpace{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: addressSpace.Name, Namespace: addressSpace.Namespace}, foundAddressSpace)
+		if err != nil && errors.IsNotFound(err) {
+			reqLogger.Info("Creating a new Address Space", "AddressSpace.Namespace", addressSpace.Namespace, "AddressSpace.Name", addressSpace.Name)
+			err = r.client.Create(context.TODO(), addressSpace)
+			if err != nil {
+				return reconcile.Result{}, err
+			}
+			return reconcile.Result{RequeueAfter: time.Second * 10}, nil
+		} else if err != nil {
+			return reconcile.Result{}, err
+		} else if !foundAddressSpace.Status.IsReady {
+			return reconcile.Result{RequeueAfter: time.Second * 10}, nil
+		} else {
+			reqLogger.Info("Found AddressSpace for UPS")
+		}
+
+		/*Address space exists and is ready*/
+
+		for _, status := range foundAddressSpace.Status.EndpointStatus {
+			if status.Name == "messaging" { //magic value
+				addressSpaceURL := status.ServiceHost
+				addressConfigMap := newAMQConfigMap(instance, addressSpaceURL)
+				err = r.client.Create(context.TODO(), addressConfigMap)
+				if err != nil && !errors.IsAlreadyExists(err) {
+					return reconcile.Result{}, err
+				}
+
+			}
+		}
+
+		//check that user exists
+		user, err := newMessagingUser(instance)
+
```

Comment:
```suggestion
```

Visually we tend to have the `if err != nil {` directly on the next line, rather than having a blank line. This will reduce the chances that someone will assume they can just put something between the function call and the error check, rather than after the error check.